### PR TITLE
Chore: made a github issues template

### PR DESCRIPTION
**Pull request by killertofus**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE /a_make_a_report.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+labels: [issue-report]
+
+body:
+  - type: checkboxes
+    id: acknowledgments
+    attributes:
+      label: Checklist
+      description:
+      options:
+        - label: I'm reporting a problem with Magmawm
```
Comment (electron271) [inline .github/ISSUE_TEMPLATE /a_make_a_report.yml:12]: should be MagmaWM

File: .github/ISSUE_TEMPLATE /a_make_a_report.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      placeholder: You can upload them using the text editor's dedicated button.
+
+  - type: input
+    id: versions
+    validations:
+      required: true
+    attributes:
+      label: OS and Magma Version
+      description: The name of your Operating System and the version shown in Magma --v
+      placeholder: Magmawm 2.3.5 (commit 81a62764, 2022-04-05) on arch, kernel 6.8.1
```
Comment (electron271) [inline .github/ISSUE_TEMPLATE /a_make_a_report.yml:52]: 2.3.5 doesnt exist and we didnt exist in 2022, best to replace with something like this:
```
MagmaWM git (commit 766d0d0)
```

**Review by electron271 — CHANGES_REQUESTED** [review]

**Comment by killertofus** [comment]
fixed, thanks for the report

File: .github/ISSUE_TEMPLATE /config.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Issue about Magmawm
```
Comment (electron271) [inline .github/ISSUE_TEMPLATE /config.yml:3]: still has incorrect spelling

**Review by electron271 — CHANGES_REQUESTED** [review]

**Comment by killertofus** [comment]
now it's fixed

File: .github/ISSUE_TEMPLATE/a_make_a_report.yml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    id: acknowledgments
+    attributes:
+      label: Checklist
+      description:
+      options:
+        - label: I'm reporting a problem with MagmaWM
+          required: true
+        - label: I've verified that I'm running **the most** recent nightly build or stable release
+          required: true
+        - label: I've looked for my problem on the [wiki](https://github.com/MagmaWM/MagmaWM/Help/)
```
Comment (HackedOS) [inline .github/ISSUE_TEMPLATE/a_make_a_report.yml:16]: the correct link for the docs/wiki https://magmawm.org/docs/

File: .github/ISSUE_TEMPLATE/a_make_a_report.yml [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+          required: true
+        - label: I've searched the [issues and pull requests](https://github.com/MagmaWM/MagmaWM/issues?q=) for similar looking reports
```
Comment (HackedOS) [inline .github/ISSUE_TEMPLATE/a_make_a_report.yml:18]: why the trailing ?q=

File: pkgbuild/PKGBUILD
Comment (HackedOS) [inline pkgbuild/PKGBUILD:1]: why is there a pkgbuild we dont intend to add one till release

File: .github/ISSUE_TEMPLATE/a_make_a_report.yml
Comment (HackedOS) [inline .github/ISSUE_TEMPLATE/a_make_a_report.yml:1]: we want a section about a how to send logs and send WAYLAND_DEBUG logs for specific software they are having issues with

**Review by HackedOS — CHANGES_REQUESTED** [review]

**Pull request closed without merging**